asteroidsyncserviced/dbusinterface.cpp

```diff
@@ -29,11 +29,16 @@
 
 DBusWatch::DBusWatch(Watch *watch, WatchesManager* wm, QObject *parent): QObject(parent), m_watch(watch), m_wm(wm)
 {
+  
```

Comment:
Please don't add empty lines which do not have semantic (logical blocks separation for example)

Reply:
I'm trying.